checks/evaluation/code_review.go

```diff
@@ -61,12 +61,8 @@ func reviewScoreForChangeset(changeset *checker.Changeset) (score reviewScore) {
 		return reviewedOutsideGithub
 	}
 
-	for i := range changeset.Commits {
-		for _, review := range changeset.Commits[i].AssociatedMergeRequest.Reviews {
-			if review.State == "APPROVED" {
-				return changesReviewed
-			}
-		}
+	if changeset.ReviewPlatform == checker.ReviewPlatformGitHub {
+		return changesReviewed
```

Comment:
This doesn't check for "if the merger is different from the committer". Tested it on a PR I created and merged here: https://github.com/spencerschrock/codereview-test.

`1 out of last 2 changesets reviewed before merge -- score normalized to 5`
The first commit was pushed straight to main and isn't marked as reviewed. The second commit was added via a PR authored and merged by me.

Reply:
There was also some additional discussion about verified commits to try to avoid faking other users in the [slack thread](https://app.slack.com/client/T019QHUBYQ3/C0235AR8N2C/thread/C0235AR8N2C-1665602612.497779)

> One caveat - note that we should be ensuring we honor verified  commits only when checking for mergedBy . It is super easy for someone to fake another user

Reply:
Yeah, that's true - verified commits, when available, should be the source of truth for the Code Review check (and requiring Verified commits could also be good as a standalone check). I do think the change is what maintainers would find useful in the short term, and we could have a separate issue for 'hardening' the Code Review check to rely on verified authors. Wdyt?

Reply:
> we could have a separate issue for 'hardening' the Code Review check to rely on verified authors

I think it's fine to harden in a followup.

But the code as written is too permissive. It counts any PR as reviewed even when the PR author is the same as the person who merges.

Reply:
Makes sense, added the check back.